dice.js

```diff
@@ -0,0 +1,9 @@
+const rollDice1 = function(numOne, numTwo){
+    var myArray = [1,2,3,4,5,6];
+
+var numOne = myArray[Math.floor(Math.random()*myArray.length)];
+var numTwo = myArray[Math.floor(Math.random()*myArray.length)];
```

Comment:
You're already using Math.random() to get random numbers from 1 to 6, you don't need an array of numbers from 1 to 6.
`let numOne = Math.floor(Math.random() * 6) + 1 ;`
works without needing an array